src/rules/useStrictRule.ts

```diff
@@ -77,4 +80,14 @@ class UseStrictWalker extends Lint.ScopeAwareRuleWalker<{}> {
             this.addFailure(this.createFailure(node.getStart(), node.getFirstToken().getWidth(), Rule.FAILURE_STRING));
         }
     }
+
+    private getFirstInModuleDeclarationsChain(node: ts.ModuleDeclaration): ts.ModuleDeclaration {
```

Comment:
since this is static, can you make this a `function` outside of this class, at the end of this file?